js/pulse.js

```diff
@@ -106,7 +106,6 @@ var newProjectForm = {
     newProjectForm.formContainer.style.display = 'none';
     newProjectForm.projectContainer.style.display = 'block';
     newProjectForm.toggleFormButton.style.transform = 'rotate(0deg)';
-    setTimeout(function(){ PulseMaker.getData(); }, 250);
```

Comment:
no need to do this again since `refresh` is already a recurring task
(see `pulse.js`, the `PulseMaker.init` method)